srcs/config/ServerBlock.cpp

```diff
@@ -0,0 +1,97 @@
+#include "ServerBlock.hpp"
+#include "utils.hpp"
+#include <ios>
+
+ServerBlock::ServerBlock() {
+    _GET_allowed = false;
+    _POST_allowed = false;
+    _DELETE_allowed = false;
+    _client_max_body_size = 100000;
+    _root_folder = "/";
+    _index_page = "index.html";
```

Comment:
Should we initialize here the `_default_error_pages` map here instead of `Config.cpp`